tests/test_router_bgp.rb

```diff
@@ -43,12 +52,14 @@ def create_bgp_vrf(asnum, vrf)
 end
 
 def setup_default
+  @device.cmd('no router bgp')
```

Comment:
@chrisvanheuveln Addressed comments - except the one about not using platform check in the test file.